ViAn/GUI/projectwidget.h

```diff
@@ -27,10 +27,7 @@ class VideoProject;
 class ProjectWidget : public QTreeWidget
 {
     Q_OBJECT
-    QTreeWidgetItem* clicked_item = nullptr;
-    QPoint* clicked_point = nullptr;
     QTreeWidgetItem* selection_parent = nullptr;
-    bool selecting = false;
```

Comment:
Couldn't find any other changes using this variable. Unused?

Reply:
Yes